pkgpanda/__init__.py

```diff
@@ -261,6 +270,19 @@ def validate_compatible(packages, roles):
                         k, v, package))
             environment[k] = package
 
+        # No conflicting sysctl values.
+        for service_name, sysctl_settings in package.sysctl.items():
+            for sysctl_var, sysctl_value in sysctl_settings.items():
+                if sysctl_map.get(sysctl_var, None) == sysctl_value:
```

Comment:
Going with the first comment.

`if sysctl_var in sysctl_map and sysctl_map[sysctl_var] != sysctl_value`

Is what I wanted to do. Since the settings are on package basis (like minuteman settings) currently and we should let other other packages add settings too. If there is a conflict, we crash. If not, we should proceed. 

If the settings were consolidated (in future?) then, this should be changed to a set and done something like 2nd comment. But leaving the sysctl (and binary capabilities next) to be managed by package owners seems to be a good idea to me, and dcos does the job of validation.